sync/syncer.go

```diff
@@ -291,149 +399,90 @@ func (s *Syncer) Txs(mb *types.Block) ([]*types.SerializableTransaction, error)
 
 	//map and sort txs
 	if len(missing) > 0 {
-		for out := range s.fetchWithFactory(TxReqFactory(missinDB), 1) {
+		for out := range s.fetchWithFactory(NewNeighborhoodWorker(s, 1, TxReqFactory(missinDB))) {
```

Comment:
what is the 1 parameter? add comment

Reply:
number of workers its clear from the param name